.github/workflows/workflow.yml

```diff
@@ -38,6 +38,11 @@ on:
         type: string
         default: ""
 
+      checkout_ref:
+        required: false
+        default: main
```

Comment:
I think you'd want this to default to the `default_branch` variable as not all repos are updated to use `main` yet.